main_test.go

```diff
@@ -31,23 +85,27 @@ func MakeTestConfig() *AppConfig {
 		panic(err)
 	}
 	pubKeys, _ := keyBag.AvailableKeys()
+	rsaKey, _ := rsa.GenerateKey(rand.Reader, 1024)
 	return &AppConfig{
 		Broker{0, 0},
 		BlockChain{
 			eos.Checksum256(chainID),
 			casinoAccName,
 			PubKeys{pubKeys[0], pubKeys[1]},
-			nil,
+			rsaKey,
 		},
-	}
+	}, &keyBag
 }
 
 func TestMain(m *testing.M) {
 	InitLogger("debug")
 	events := make(chan *broker.EventMessage)
-	listener := broker.NewEventListener(":8888", events)
-	f := &bytes.Buffer{}
-	a = NewApp(eos.New("https://api.daobet.org"), listener, events, f, MakeTestConfig())
+	listener := new(EventListenerMock)
+	f := &SafeBuffer{}
+	appCfg, keyBag := MakeTestConfig()
+	bc := eos.New("https://api.daobet.org")
```

Comment:
after that it's not unit test anymore :)
Don't use external resources in unit tests. For that we will use integration tests.